certbot/certbot/_internal/auth_handler.py

```diff
@@ -4,6 +4,7 @@
 import time
 
 import zope.component
+from josepy.jwk import JWK # pylint: disable=unused-import
```

Comment:
why do we need this import?

Reply:
I have a vague memory of either pylint or mypy complaining at me at some point so I added it. But maybe I  got confused, it seems like I can get rid of it now.

To be honest the Sphinx docstrings do my head in. I am never quite sure what the syntax is, whether any of our linters are checking them, or whether there's any point writing them in `_internal` packages now that we have type annotations.

Reply:
That's fair. We had those well before we had `_internal` at all; I usually try to stick them in any public method, even in `_internal` classes, because I find a short description with a couple more words than a variable name helpful when reading back through.

Reply:
Let's try removing it again and see if something yells? Might have been an interim stage.

Reply:
Oh, does mypy need a package available to do type checking? That seems weird...

Reply:
I did remove it in https://github.com/certbot/certbot/pull/8514/commits/651cc7cd071b1551729a421cb3c2bb234b8dba90 and it's not complaining :tada: .